xml/System.Reflection/FieldInfo.xml

```diff
@@ -670,7 +670,7 @@
 
  To obtain the normal, unmodified type from a modified type, use <xref:System.Type.UnderlyingSystemType%2A>.
 
- This method is provided for designers of managed compilers. For more information on custom modifiers, see classes in the <xref:System.Runtime.CompilerServices> namespace. Also see the metadata specification in Partition II of the [Common Language Infrastructure (CLI) documentation](https://www.ecma-international.org/publications-and-standards/standards/ecma-335/).  
+ This method is provided for designers of managed compilers. For more information on custom modifiers, see classes in the <see cref="N:System.Runtime.CompilerServices"/> namespace. Also see the metadata specification in Partition II of the [Common Language Infrastructure (CLI) documentation](https://www.ecma-international.org/publications-and-standards/standards/ecma-335/).  
```

Comment:
I can't offer a full suggestion to replace the remarks with plain xml. For that reason, my suggestion for this case will be to move it back to xref, because it's throwing a warning in CI.

```suggestion
 This method is provided for designers of managed compilers. For more information on custom modifiers, see classes in the <xref:System.Runtime.CompilerServices> namespace. Also see the metadata specification in Partition II of the [Common Language Infrastructure (CLI) documentation](https://www.ecma-international.org/publications-and-standards/standards/ecma-335/).  
```